File: src/main/java/com/bankone/auth/dto/LoginResponse.java
package com.bankone.auth.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class LoginResponse {
    private String accessToken;
    private String tokenType;
    private Long expiresIn;
    private List<String> roles;
    private String username;
    private String firstName;
    private String lastName;
    private String email;
}

File: BankOne-Frontend/src/app/core/models/login-response.ts
export interface LoginResponse {
  accessToken: string;
  tokenType: string;
  expiresIn: number;
  roles: string[];
  accesses?: string[];
  customerId?: number | null;
  portalUser?: boolean;
  username: string;
  firstName?: string;
  lastName?: string;
  email?: string;
}
